src/SettingsMigrator/Resources/xlf/Resources.cs.xlf

```diff
@@ -41,13 +41,18 @@
         <note />
       </trans-unit>
       <trans-unit id="ValidUsage">
-        <source>Valid usage: SettingsMigrator.exe &lt;Full path to testsettings file or runsettings file to be migrated&gt;
-Example: SettingsMigrator.exe  E:\MyTest\MyTestSettings.testsettings 
-Example: SettingsMigrator.exe  E:\MyTest\MyOldRunSettings.runsettings</source>
-        <target state="translated">Platné použití: SettingsMigrator.exe &lt;kompletní cesta k souboru testsettings nebo runsettings, který se má migrovat&gt;
+        <source>Valid usage:
```

Comment:
These resources update missed in https://github.com/Microsoft/vstest/pull/1620
/cc @singhsarab  @kaadhina